modtools/create-unit.lua

```diff
@@ -25,7 +25,6 @@ local usage = [====[
 modtools/create-unit
 ====================
 Creates a unit.  Usage::
-
```

Comment:
? @ lethosor is that an automated message or can you actually see what I'm doing as I do it?